NuimoSDK/NuimoBluetoothController.cs

```diff
@@ -153,12 +153,12 @@ private bool ReadCharacteristicValue(Guid characteristicGuid, Action<byte[]> onV
             return !readResult.IsCanceled;
         }
 
-        public async void DisplayLedMatrixAsync(NuimoLedMatrix matrix, double displayInterval = 2.0, int options = 0)
+        public async void DisplayLedMatrixAsync(NuimoLedMatrix matrix, double displayInterval = 2.0, NuimoLedMatrixWriteOption options = NuimoLedMatrixWriteOption.None)
         {
             if (ConnectionState != NuimoConnectionState.Connected) return;
 
-            var withFadeTransition   = (options & (int) NuimoLedMatrixWriteOption.WithFadeTransition)   != 0;
-            var writeWithoutResponse = (options & (int) NuimoLedMatrixWriteOption.WithoutWriteResponse) != 0;
+            var withFadeTransition   = options.HasFlag(NuimoLedMatrixWriteOption.WithFadeTransition);
+            var writeWithoutResponse = options.HasFlag(NuimoLedMatrixWriteOption.WithoutWriteResponse);
```

Comment:
@larsblumberg thank you for the feedback.

I could do this but actually I think that the API design could be improved here. With the options as they are now, in my opinion you mixed settings having two different purposes:
- WithFadeTransition has an effect on the LED effect
- WithoutWriteResponse has an effect on whether we should wait for the response

I would do the following:
- create a LEDMatrixCommand class with the following properties:
-- NuimoLedMatrix Matrix
-- double DisplayInterval
-- bool WithFadeTransition
- modify DisplayLedMatrixAsync() to have the following parameters:
-- LEDMatrixCommand command
-- bool WithoutWriteResponse

This separates the LED effect parameters better from the way how the program flow should be (with or without waiting).

What do you think?